build_runner/pubspec.yaml

```diff
@@ -0,0 +1,34 @@
+name: build_runner
+version: 0.1.0
+description: Tools to write binaries that run builders.
+author: Dart Team <misc@dartlang.org>
+homepage: https://github.com/dart-lang/build
+
+environment:
+  sdk: '>=1.9.1 <2.0.0'
+
+dependencies:
+  analyzer: ">=0.27.1 <0.30.0"
+  build: ^0.6.0
+  build_barback: 0.0.1
```

Comment:
Unfortunately this one still needs build_barback for now since it uses `BarbackResolvers` as a default implementation. Once we can find another way to provide a reasonable default without code_transformers we can break this dependency